.github/workflows/linux-install.yaml

```
@@ -0,0 +1,54 @@
+name: Linux-Install
+run-name: Linux-Install
+on:
+  pull_request:
+    paths-ignore:
+    - README.md
+    - 'docs/**'
+    types: [ opened, reopened, synchronize ]
+    
+jobs:
+   Kokkos-040200-OpenMPI:
+    env:
+      KOKKOS_SRC: ${{ github.workspace }}/_deps/kokkos
+      KOKKOS_BUILD: ${{ github.workspace }}/_deps/kokkos-build
+      KOKKOS_INSTALL: ${{ github.workspace }}/_deps/kokkos-install
+      COMM_SRC: ${{ github.workspace }}
+      COMM_BUILD: ${{ github.workspace }}/build
+      COMM_INSTALL: ${{ github.workspace }}/install
+      COMM_INSTALL_BUILD: ${{ github.workspace }}/install-build
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Install MPI
+        run: |
+          sudo apt-get update && sudo apt-get install -y libopenmpi-dev openmpi-bin cmake
+          which mpirun
+          mpirun --version
+          which mpicxx
+          mpicxx --version
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: List files in the repository
+        run: |
+          ls ${{ github.workspace }}
+      - name: Build Kokkos
+        run: |
+          git clone https://github.com/kokkos/kokkos.git "$KOKKOS_SRC"
+          cd "$KOKKOS_SRC" && git checkout 4.2.00
+          cmake -S "$KOKKOS_SRC" -B "$KOKKOS_BUILD" -DCMAKE_INSTALL_PREFIX="$KOKKOS_INSTALL" -DCMAKE_BUILD_TYPE=Release -DKokkos_ENABLE_SERIAL=ON -DKokkos_ENABLE_OPENMP=ON -DKokkos_ARCH_NATIVE=ON -DCMAKE_CXX_COMPILER=mpicxx
```

```suggestion
          cmake -S "$KOKKOS_SRC" -B "$KOKKOS_BUILD" -DCMAKE_INSTALL_PREFIX="$KOKKOS_INSTALL" -DCMAKE_BUILD_TYPE=Release -DKokkos_ENABLE_SERIAL=ON -DKokkos_ENABLE_OPENMP=ON -DCMAKE_CXX_COMPILER=mpicxx
```

We do not know if GitHub actions run on homogeneous machines, so do not over-optimize for CI-robustness.
Do we really need both `SERIAL` and `OPENMP`?

---

Just making sure things work when two backends are enabled. Could be OpenMP or Threads.

---

I'm okay with removing Kokkos_ARCH_NATIVE